--[[
	SplashTexts.luau
	Random splash texts shown on the title screen (like Minecraft!)
	
	Add your own phrases here - one will be randomly chosen each time
	someone loads the game.
]]

return {
	-- Fun phrases
	"Now with 100% more blocks!",
	"Place. Build. Create!",
	"Also try Minecraft!",
	"Blox on blox on blox!",
	"Now in 3D!",
	"Contains real pixels!",
	"As seen on screen!",
	"Wow!",
	"Limited edition!",
	"It's a feature, not a bug!",
	"Made with love!",
	"Building dreams!",
	"One block at a time!",
	"Stack 'em high!",
	"Touch grass... in game!",
	"Beta tested!",
	"Community approved!",
	"Creatively creative!",
	"No creepers here!",
	"Safe for builders!",
	"Infinite possibilities!",
	"Your world, your rules!",
	"Block by block!",
	"Building the future!",
	"Stairway to heaven included!",
	"Fences make good neighbors!",
	"Let there be light!",
	"Torches sold separately!",
	"Glass half full!",
	"Door-able!",
	"Sand falls down!",
	"Gravity optional!",
	"Save early, save often!",
	"Multi-save support!",
	"Lanterns light the way!",
}
